### PR TITLE
sp.live.nicovideo.jp  

### LINKED ISSUE #108533: sp.live.nicovideo.jp
### Issue URL (Ads)
 [https://sp.live.nicovideo.jp/watch/lv335023046](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsp.live.nicovideo.jp%2Fwatch%2Flv335023046)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.3.2
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese, <br/>AdGuard Chinese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #108533

### CODE REVIEW

File: MobileFilter/sections/specific_web.txt [hunk 1]
```diff
@@ -27,7 +27,6 @@ scmp.com,theinventory.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalo
 ! ##.ads
 cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m.ura.news,siirtpress.com,up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org,m-economictimes-com.cdn.ampproject.org,m.winfuture.de##.ads
 !
-sp.live.nicovideo.jp##aside[class*="banner-ad"]
```
Comment (SKEIDs) [inline MobileFilter/sections/specific_web.txt]: same rule
https://github.com/SKEIDs/AdguardFilters/blob/7105b91723ca29dfe620e782830c0fb9603ec98c/JapaneseFilter/sections/specific.txt#L2478

Comment (SKEIDs) [inline MobileFilter/sections/specific_web.txt]: same rule https://github.com/AdguardTeam/AdguardFilters/pull/107473/commits/7105b91723ca29dfe620e782830c0fb9603ec98c

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 2]
```diff
@@ -15,6 +15,7 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+sp.live.nicovideo.jp##a[class*="app-download-banner"]
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/mobile-app.txt:18]: Should be removed as per https://github.com/AdguardTeam/AdguardFilters/pull/107473 but alternatively you can close the old PR and merge the rules to this PR.

File: MobileFilter/sections/specific_web.txt [hunk 3]
```diff
@@ -27,6 +27,7 @@ scmp.com,theinventory.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalo
 ! ##.ads
 cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m.ura.news,siirtpress.com,up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org,m-economictimes-com.cdn.ampproject.org,m.winfuture.de##.ads
 !
+sp.live.nicovideo.jp##div[class*="footer-banner-ads"]
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:30]: `live.nicovideo.jp##div[class^="___footer-banner-ads"]`

File: SocialFilter/sections/specific.txt [hunk 4]
```diff
@@ -49,6 +49,7 @@ zamaninvarken.com,kredi.biz.tr,kriptoradar.com,morlevha.com,korsanedebiyat.com,o
...
 !
+sp.live.nicovideo.jp##button[class*="social-sharing-button"]
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:52]: `live.nicovideo.jp##button[class^="___social-sharing-button"]`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
On my end `live.nicovideo.jp##div[class^="___overlay-banner-ad"]` is also needed. Not sure about the top banner, looks to be temporal notification.

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 2 above)
Comment (SKEIDs) [reply AnnoyancesFilter/sections/mobile-app.txt]: ok

**Comment by SKEIDs** [comment]
> On my end `live.nicovideo.jp##div[class^="___overlay-banner-ad"]` is also needed. Not sure about the top banner, looks to be temporal notification.

done

**Review by Yuki2718 — APPROVED** [review]

**Comment by SKEIDs** [comment]
@zloyden could you merge this ?

**Review by zloyden — APPROVED** [review]

**Pull request merged**